ReactAndroid/src/main/java/com/facebook/react/ReactActivityDelegate.java

```diff
@@ -111,44 +105,19 @@ protected void onResume() {
   }
 
   protected void onDestroy() {
-    if (mReactRootView != null) {
-      mReactRootView.unmountReactApplication();
-      mReactRootView = null;
-    }
-    if (getReactNativeHost().hasInstance()) {
-      getReactNativeHost().getReactInstanceManager().onHostDestroy(getPlainActivity());
-    }
+    mReactDelegate.onHostDetroy();
```

Comment:
Minor misspelling here of "Destroy" --- should be `mReactDelegate.onHostDestroy();`